packages/rest/src/router/routing-table.ts

```diff
@@ -127,6 +126,10 @@ export class RoutingTable {
       );
     }
     this._routes.push(route);
+    // Add the route to the trie
+    const path = route.path.startsWith('/') ? route.path : `/${route.path}`;
+    const verb = route.verb.toLowerCase() || 'get';
+    this._router.on(`/${verb}${path}`, () => {});
```

Comment:
Can we store `route` on the function passed to `this._router.on` to avoid costly lookup (iteration through `this._routes` later on? E.g.

```ts
// store
this._router.on(`/${verb}${path}`, Object.assign(() => {}, {route}));

// pick up - I hope that's the Wayfarer API
const found = this._router.match(path);
const route = found.route;
```